draft-ietf-masque-h3-datagram.md

```diff
@@ -82,7 +82,7 @@ version of the CONNECT method, or adding unreliable delivery to WebSockets
 {{?RFC6455}}.
 
 In {{datagrams}}, this document describes HTTP Datagrams, a convention that
-supports the bidirectional and possibly multiplexed exchange of data inside an
```

Comment:
I find the use of "possibly" to be awkward. The protocol supports the feature, it doesn't possibly support it.